### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lipcai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,35 @@ Our app support interactivity between graphs. Detailed instruction as follows:
...
+We welcome all feedback and contributions. If you are interested in contributing to our app, please read our [contributing document](https://github.com/UBC-MDS/movielover_py/blob/main/CONTRIBUTING.md) and our [code of conduct](https://github.com/UBC-MDS/movielover_py/blob/main/CODE_OF_CONDUCT.md). Our [license](https://github.com/UBC-MDS/movielover_py/blob/main/LICENSE) outlines our permissions and copyright information for users. You are welcomed to raise new ideas and report any existing bugs. 
+
+### How to install and run locally
+
+To run the dashboard locally, it is recommeded to use a virtual environment like [venv](https://docs.python.org/3/library/venv.html) or [Anaconda](https://www.anaconda.com/). For simplicity, we could demonstrate the installiation process with venv.
```
Comment (zjr-mds) [inline README.md]:
```suggestion
### How to install and run locally

To run the dashboard locally, it is recommended to use a virtual environment like [venv](https://docs.python.org/3/library/venv.html). The installation process with venv is demonstrated below.
```

File: README.md [hunk 2]
```diff
@@ -38,7 +38,35 @@ Our app support interactivity between graphs. Detailed instruction as follows:
...
+
+### Set up
+
+Run the following command at the root directory of the project:
```
Comment (zjr-mds) [inline README.md]:
```suggestion
Run the following command at the **root** directory of the project in the terminal:
```

File: README.md [hunk 3]
```diff
@@ -38,7 +38,35 @@ Our app support interactivity between graphs. Detailed instruction as follows:
...
+pip install -r requirements.txt
+```
+
+### Run the dashboard
+
+```
+python src/app.py
+```
+
+The dashboard could then be accessed locally in <localhost:8050>, and you are good to go!
```
Comment (zjr-mds) [inline README.md]:
```suggestion
The dashboard could then be accessed locally in <localhost:8050>. 

*Tips for dashboard first-time users: you might see something similar to `Dash is running on http://127.0.0.1:8050/` after running the command above, by simply copying and pasting `http://127.0.0.1:8050/` to a browser, you will be able to view the dashboard. If you keep the dashboard open, the page will automatically refresh whenever you make additional error-free changes.* 
```

**Review by zjr-mds — CHANGES_REQUESTED** [review]

**Review by zackt113 — CHANGES_REQUESTED** [review]
Please build a section link on the top of the README file on the things you newly added, so everything is consistent! Thank you! Just like that * [Visualization App Purpose](#visualization-app-purpose)

**Review by zackt113 — APPROVED** [review]
LGTM

**Review by zjr-mds — APPROVED** [review]

**Pull request merged**